policies/coordinated-write-permissions.md

```diff
@@ -0,0 +1,9 @@
+# Write Permissions for Coordinated Package Repositories
+
+This policy describes the rules and expectations for receiving write permissions for Coordinated Packages (including the Astropy Core Package).
+
+Github write privileges are granted when someone is given a relevant named role. In most cases this is package (or sub-package) maintainer, although other roles (for example, the astropy.org web page maintainer) may also come with write permissions for a specific named repository.  Occasional *temporary* write (or higher) permissions may be grant for specific tasks (for example, a maintainer may be given admin rights to a repo to configure CI for the first time or similar), be such permissions must be done temporarily.
```

Comment:
typo: be such permission - > but

Reply:
```suggestion
Github write privileges are granted when someone is given a relevant named role. In most cases this is package (or sub-package) maintainer, although other roles (for example, the astropy.org web page maintainer) may also come with write permissions for a specific named repository.  Occasional *temporary* write (or higher) permissions may be grant for specific tasks (for example, a maintainer may be given admin rights to a repo to configure CI for the first time or similar), but such permissions must be done temporarily.
```

Reply:
this conflicted with @nstarman's change, but I updated it in https://github.com/astropy/astropy-project/pull/99/commits/950b016fe55775b4f6fd1da6d818167df1cf2ab1